bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -318,6 +318,75 @@ impl Script {
         crate::Amount::from_sat(sats)
     }
 
+    /// Count the sigops for this Script using accurate counting.
```

Comment:
```suggestion
    /// Counts the sigops for this Script using accurate counting.
```

For the follow up :)

Reply:
Bit picky I know.